index.js

```diff
@@ -18,6 +19,12 @@ updateNotifier({pkg}).notify()
 main(parseArgs())
 
 function main (argv) {
+  const shell = argv['shell-auto-fallback']
+  if (shell || shell === '') {
```

Comment:
`requireArg` is `false`! So sometimes the option will return null/undefined/false/whatever yargs returns for a non-provided option. But `''` is falsy, so I do need to check.